src/components/Popups/ReplyField/ReplyForm.tsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import Button from 'box-ui-elements/es/components/button';
+import PrimaryButton from 'box-ui-elements/es/components/primary-button';
+import { EditorState } from 'draft-js';
+import { FormikProps } from 'formik';
+import isEmpty from 'lodash/isEmpty';
+import { FormattedMessage, useIntl } from 'react-intl';
+import ReplyField from './ReplyFieldContainer';
+import { FormValues, Props as OtherProps } from './ReplyFormContainer';
+import messages from '../messages';
+
+type Props = OtherProps & FormikProps<FormValues>;
+
+const ReplyForm = ({
+    errors,
+    handleSubmit,
+    isPending,
+    onCancel,
+    onChange,
+    setFieldValue,
+    values,
+}: Props): JSX.Element => {
+    const intl = useIntl();
+    const hasErrors = !isEmpty(errors);
+    const handleChange = (editorState: EditorState): void => {
+        const text = editorState.getCurrentContent().getPlainText();
+        onChange(text);
+        setFieldValue('editorState', editorState);
+    };
+
+    return (
+        <form className="ba-Popup-form" data-testid="ba-Popup-form" onSubmit={handleSubmit}>
+            <div className="ba-Popup-main">
+                <ReplyField
+                    className="ba-Popup-field"
+                    data-testid="ba-Popup-field"
+                    editorState={values.editorState}
+                    isDisabled={isPending}
+                    onChange={handleChange}
+                    placeholder={intl.formatMessage(messages.fieldPlaceholder)}
+                />
+            </div>
+            <div className="ba-Popup-footer">
+                <Button data-testid="ba-Popup-cancel" isDisabled={isPending} onClick={onCancel} type="button">
+                    <FormattedMessage {...messages.buttonCancel} />
+                </Button>
+                <PrimaryButton data-testid="ba-Popup-submit" isDisabled={hasErrors || isPending} type="submit">
```

Comment:
Does this still have the post button disabled until the text field has content?

Reply:
Yup!